libs/deal-ticket/src/components/deal-ticket/side-selector.tsx

```diff
@@ -14,14 +15,30 @@ export const SideSelector = ({ value, onSelect }: SideSelectorProps) => {
     { label: t('Short'), value: Schema.Side.SIDE_SELL },
   ];
 
+  const [side, setSide] = useState(value);
```

Comment:
chore (non blocking): not sure why we need new state here. Cant we just pass value?

```
type={toggleType(value)}
```